src/modules/licences/controllers/licences.js

```diff
@@ -38,8 +38,24 @@ const getValidLicenceDocumentByDate = async request => {
   return documentsService.getValidDocumentOnDate(licence.licenceNumber, date);
 };
 
+const getLicencesWithoutChargeVersions = async () => {
+  const licences = await licencesService.getLicencesWithoutChargeVersions();
+
+  const promises = licences.map(licence => {
+    return documentsService.getValidDocumentOnDate(licence.licenceNumber, licence.startDate)
```

Comment:
Small issue. This fails if a licence doesn't have any corresponding document records. 

The API then returns:
```
message: Current or superseded document not found for AN/xxx/xxxx/xxx on 2020-03-13, stack: NotFoundError: Current or superseded document not found for AN/xxx/xxxx/xxx on 2020-03-13
```